src/appleseed.python/bindproject.cpp

```diff
@@ -305,7 +305,9 @@ void bind_project()
     bpy::enum_<ProjectFileReader::Options>("ProjectFileReaderOptions")
         .value("Defaults", ProjectFileReader::Defaults)
         .value("OmitReadingMeshFiles", ProjectFileReader::OmitReadingMeshFiles)
-        .value("OmitProjectFileUpdate", ProjectFileReader::OmitProjectFileUpdate);
+        .value("OmitProjectFileUpdate", ProjectFileReader::OmitProjectFileUpdate)
+        .value("OmitSearchPaths", ProjectFileReader::OmitSearchPaths)
```

Comment:
We want to use the search paths in the "parent" project, not the ones in the project we are importing.

What happens is that we copy the search path of the parent project into the new project and then pass OmitSearchPaths to ignore the search paths in the project we are reading.